### PR TITLE
Add symfony 5.3

### PR DESCRIPTION
**Pull request by Chris53897**
| Q             | A
| ------------- | ---
| Branch?       | main
| License       | MIT

Bump symfony 5.2 to 5.3

Bump >=8.0 to >=8.0.2
Symfony 6 is using this, so i guess they avoid some bugs in 8.0.0 and 8.0.1
https://github.com/symfony/symfony/blob/6.0/composer.json

### CODE REVIEW

**Comment by Chris53897** [comment]
Argh. i just want a title or whatever is needed so the UnitTests will start :(

**Comment by soyuka** [comment]
the commit message should respsect the conventional commits spec: https://www.conventionalcommits.org/en/v1.0.0/

**Comment by vincentchalamon** [comment]
You should also update the recipes: `composer sync-recipes --force`
Some config files will be updated, please check for some useless files that may be deleted.

File: api/config/packages/framework.yaml [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
-        handler_id: null
-        cookie_secure: auto
-        cookie_samesite: lax
 
     #esi: true
     #fragments: true
     php_errors:
         log: true
+
+when@test:
```
Comment (vincentchalamon) [inline api/config/packages/framework.yaml:16]: You should remove the api/config/packages/test/framework.yaml file

File: api/config/packages/routing.yaml [hunk 2]
```diff
@@ -5,3 +5,8 @@ framework:
         # Configure how to generate URLs in non-HTTP contexts, such as CLI commands.
         # See https://symfony.com/doc/current/routing.html#generating-urls-in-commands
         #default_uri: http://localhost
+
+when@prod:
```
Comment (vincentchalamon) [inline api/config/packages/routing.yaml:9]: You should remove the api/config/packages/prod/routing.yaml file

File: api/config/packages/twig.yaml [hunk 3]
```diff
@@ -1,2 +1,6 @@
 twig:
     default_path: '%kernel.project_dir%/templates'
+
+when@test:
```
Comment (vincentchalamon) [inline api/config/packages/twig.yaml:4]: You should remove the api/config/packages/test/twig.yaml file

File: api/config/routes/framework.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+when@dev:
```
Comment (vincentchalamon) [inline api/config/routes/framework.yaml:1]: You should remove the api/config/routes/dev/framework.yaml & twig.yaml files

**Review by vincentchalamon — CHANGES_REQUESTED** [review]

**Pull request merged**